src/phoenix/server/api/input_types/SpanSort.py

```diff
@@ -68,24 +102,13 @@ def __call__(
                 evals=evals,
             )
         else:
-            get_sort_key_value = partial(
-                _get_column_value,
-                span_column=self.col or SpanColumn.startTime,
-            )
+            NotImplementedError("deprecated")
```

Comment:
Not sure I follow the error.  Can you add more description

Reply:
It's removing the code path that extracts a span column for filter (`_get_column_value`). In other words, nothing in our code base should reach this point after we have moved to sql.

Reply:
will clarify in the error message